<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>442</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>590</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Run Simulation</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>Simulate interpo error mat</string>
              </attribute>
              <widget class="QWidget" name="widget" native="true">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>10</y>
                 <width>501</width>
                 <height>251</height>
                </rect>
               </property>
               <widget class="QGroupBox" name="groupBox">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>271</width>
                  <height>251</height>
                 </rect>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>n_grid_points</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>n</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0" colspan="3">
                  <widget class="QPushButton" name="btn_simulate">
                   <property name="text">
                    <string>Simulate</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" colspan="2">
                  <widget class="QDoubleSpinBox" name="txt_n">
                   <property name="decimals">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <double>999999999999999983222784.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" colspan="2">
                  <widget class="QDoubleSpinBox" name="txt_n_r">
                   <property name="decimals">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <double>10000000000000000000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1" colspan="2">
                  <widget class="QDoubleSpinBox" name="txt_cb">
                   <property name="decimals">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <double>1000000000000000000000.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>n_r</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1" colspan="2">
                  <widget class="QDoubleSpinBox" name="txt_n_grid_points">
                   <property name="decimals">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <double>1000000000000000019884624838656.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>cb</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>n_states</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1" colspan="2">
                  <widget class="QDoubleSpinBox" name="txt_n_states">
                   <property name="decimals">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <double>9999999999999999635896294965248.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QGroupBox" name="groupBox_5">
                <property name="geometry">
                 <rect>
                  <x>280</x>
                  <y>10</y>
                  <width>221</width>
                  <height>231</height>
                 </rect>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout_9">
                 <item row="0" column="0">
                  <widget class="QPlainTextEdit" name="simul_result"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Dynamic Program</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>n</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" colspan="3">
                  <widget class="QSpinBox" name="txt_dp_n"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>T</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1" colspan="3">
                  <widget class="QSpinBox" name="txt_dp_T"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="lbl_dp_nu">
                   <property name="text">
                    <string>n_u</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1" colspan="3">
                  <widget class="QSpinBox" name="txt_dp_nu"/>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="lbl_dp_nq">
                   <property name="text">
                    <string>n_q</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1" colspan="3">
                  <widget class="QSpinBox" name="txt_dp_nq"/>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="lbl_dp_npoints">
                   <property name="text">
                    <string>n_points</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1" colspan="3">
                  <widget class="QSpinBox" name="txt_dp_npoints"/>
                 </item>
                 <item row="5" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>B</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="QToolButton" name="btn_B">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="2">
                  <widget class="QLabel" name="lbl_B_path">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="3">
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>50</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Met Type</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="3">
                  <widget class="QPlainTextEdit" name="dp_result"/>
                 </item>
                 <item row="0" column="1" colspan="2">
                  <widget class="QComboBox" name="dp_combo_met">
                   <property name="editable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="btn_run_dp">
                <property name="text">
                 <string>Run</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_status">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionexit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
